pkg/gameservers/localsdk_test.go

```diff
@@ -48,4 +48,8 @@ func TestLocal(t *testing.T) {
 	close(stream.msgs)
 
 	wg.Wait()
+
+	gs, err := l.GetGameServer(ctx, e)
+	assert.Nil(t, err)
+	assert.Equal(t, "local", gs.GetObjectMeta().Name)
```

Comment:
You can equal a full object with testity.

Reply:
Good call. Changed.